src/discussions/posts/post-editor/PostEditor.jsx

```diff
@@ -301,6 +307,25 @@ function PostEditor({
               value={values.title}
             />
             <FormikErrorFeedback name="title" />
+            {(editExisting && userIsPrivileged) && (
```

Comment:
I think we should add another check here, which is that the current user isn't the author of the post. 

i.e. if someone is editing their own post this shouldn't show up.